liferay-page-management/liferay-portal
# LPS-185405 | Create common component for "Copy To" in Documents and Media

Motivation
=======
We need to [apply design to "Copy To" page for documents](https://issues.liferay.com/browse/LPS-185405) and [apply design to "Copy To" page for folders](https://issues.liferay.com/browse/LPS-184966). As the behaviour and the components of the page are similar, we decided to create a common component to unify them.

Proposed Solution
========
Creation of a React component to unify the "Copy To" page for items of Documents and Media, both folders and file entries.

Steps to Verify
========

Under `feature.flag.LPS-182512`

Verify copy folder:
1. Content & Data
2. Documents and Media
3. Create a Folder named "Folder"
4. Click on the three dots on the "Folder" card > Copy To
5. Select > Liferay DXP > Select in "Provided by Liferay"
6. Copy
7. View that "Folder" is inside of "Provided by Liferay"

Verify copy folder:
1. Content & Data
2. Documents and Media
3. Click the plus (:heavy_plus_sign:)  button on the top right corner > File Upload
4. Upload a file
6. Click on the three dots on the uploaded file card > Copy To
7. Select > Select in "Provided by Liferay"
8. Copy
9. View that the uploaded file is inside of "Provided by Liferay"

Screenshots
========
![image](https://github.com/liferay-echo/liferay-portal/assets/40394495/1b4b0635-6d37-4402-81ab-78973bf96bcd)



**Jira Ticket:** https://issues.liferay.com/browse/LPS-185405
